File: src/Listas2/pratica/Questao01.java
package Listas2.pratica;

import java.util.Scanner;

class Emergencia {

    String nome, telefone, local, dataHora, natureza, tipo, status, feitoNoCel, coordenadas;

    public void chamarEmergencia(Scanner sc) {
        nome = sc.nextLine();
        telefone = sc.nextLine();
        local = sc.nextLine();
        dataHora = sc.nextLine();
        natureza = sc.nextLine();
        tipo = sc.nextLine();
        status = sc.nextLine();
        feitoNoCel = sc.nextLine();

        if (feitoNoCel.equalsIgnoreCase("sim")) {
            feitoNoCel = "Sim";
            coordenadas = sc.nextLine();
        } else {
            feitoNoCel = "Não";
            coordenadas = null;
        }
    }


    public void mostrarEmergencias(int quant) {
        if (quant != 1) {
            System.out.println();
        }
        System.out.println("Emergência #" + quant);
        System.out.println();
        System.out.println("--- Informações da Emergência ---");
        System.out.println();
        System.out.println("Nome do Chamador: " + nome);
        System.out.println("Telefone: " + telefone);
        System.out.println("Local da Emergência: " + local);
        System.out.println("Data/Hora do Relato: " + dataHora);
        System.out.println("Natureza da Emergência: " + natureza);
        System.out.println("Tipo de Resposta: " + tipo);
        System.out.println("Status da Resposta: " + status);
        System.out.println("Chamada via celular: " + feitoNoCel);
        if (feitoNoCel.equalsIgnoreCase("sim")) {
            System.out.println("Coordenadas GPS: " + coordenadas);
        }

    }
}

public class Questao01 {

    public static void main(String[] args) {


        //Emergencia emergencia = new Emergencia();

        Scanner sc = new Scanner(System.in);
        int quant = sc.nextInt();
        sc.nextLine();

        for (int i = 1; i <= quant; i++) {
            Emergencia emergencia = new Emergencia();
            emergencia.chamarEmergencia(sc);
            emergencia.mostrarEmergencias(i);
        }
        sc.close();
    }
}

File: Listas C++/Lista 2/questao01.cpp
#include <iostream>
#include <string>

using namespace std;

class Emergencia {
public:
    string nome, telefone, local, dataHora, natureza, tipo, status, feitoNoCel, coordenadas;

    void chamarEmergencia() {
        getline(cin, nome);
        getline(cin, telefone);
        getline(cin, local);
        getline(cin, dataHora);
        getline(cin, natureza);
        getline(cin, tipo);
        getline(cin, status);
        getline(cin, feitoNoCel);

        if (feitoNoCel == "sim" || feitoNoCel == "SIM" || feitoNoCel == "Sim") {
            feitoNoCel = "Sim";
            getline(cin, coordenadas);
        } else {
            feitoNoCel = "Não";
            coordenadas = "";
        }
    }

    void mostrarEmergencias(int quant) {
        if (quant != 1) {
            cout << endl;
        }

        cout << "Emergência #" << quant << endl;
        cout << endl;
        cout << "--- Informações da Emergência ---" << endl;
        cout << endl;
        cout << "Nome do Chamador: " << nome << endl;
        cout << "Telefone: " << telefone << endl;
        cout << "Local da Emergência: " << local << endl;
        cout << "Data/hora do Relato: " << dataHora << endl;
        cout << "Natureza da Emergência: " << natureza << endl;
        cout << "Tipo de Resposta: " << tipo << endl;
        cout << "Status da Resposta: " << status << endl;
        cout << "Chamada via celular: " << feitoNoCel << endl;

        if (feitoNoCel == "Sim") {
            cout << "Coordenadas GPS: " << coordenadas << endl;
        }
    }
};

int main() {
    int quant;
    cin >> quant;
    cin.ignore();

    for (int i = 1; i <= quant; i++) {
        Emergencia emergencia;
        emergencia.chamarEmergencia();
        emergencia.mostrarEmergencias(i);
    }

    return 0;
}